cmd/kubeadm/app/cmd/phases/markmaster.go

```diff
@@ -75,7 +75,7 @@ func NewCmdMarkMaster() *cobra.Command {
 			client, err := kubeconfigutil.ClientSetFromFile(kubeConfigFile)
 			kubeadmutil.CheckErr(err)
 
-			err = markmasterphase.MarkMaster(client, internalcfg.NodeName)
+			err = markmasterphase.MarkMaster(client, internalcfg.NodeName, true)
```

Comment:
Does this means a master taint is added by default?

Shall we add an option `NoTaintMaster` for this as well?

Reply:
I suppose we might as well, I'll do this too.